tests/infrastructure/instance_types/test_common_vm_preference.py

```
@@ -54,14 +61,27 @@ def start_vm_with_cluster_preference(client, preference_name, namespace_name):
-                preference_name=preference_name,
-                namespace_name=namespace.name,
-            )
+        if VIRTIO in preference_name or any(
+            suffix in preference_name for suffix in SUPPORTED_CPU_ARCHITECTURES - {cluster_arch}
+        ):
+            skipped_preferences.append(preference_name)
+        else:
+            runnable_preferences.append(preference_name)
+
+    if skipped_preferences:
+        LOGGER.info(f"Skipping preferences not applicable to cluster arch {cluster_arch}: {skipped_preferences}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)